src/service.ts

```diff
@@ -81,9 +90,14 @@ export default class ElectronWorkerService implements Services.ServiceInstance {
   }
 }
 
-async function callApi(bridgePropName: string, args: unknown[], done: (result: unknown) => void) {
+async function callApi(
+  contextBridgeNotAvailableError: string,
+  bridgePropName: string,
+  args: unknown[],
+  done: (result: unknown) => void,
+) {
   if (window.wdioElectron === undefined) {
-    throw new Error(`ContextBridge not available for invocation of "${bridgePropName}" API`);
+    throw new Error(contextBridgeNotAvailableError);
```

Comment:
I think you can just read this error from the constant instead of passing it in?

Reply:
nope, this command is run within the renderer process